panic in -o parsing

❯ isctl get compute physicalsummary --top 2 -o jsonpath '$[*].Moid'
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.resultHandler({0x108ca4640, 0x1400009a420}, 0x107cf65ff?, {0x0?, 0x0}, {0x140005a0430?, 0x14000145ae8?, 0x10532a844?})
	/go/src/github.com/cgascoig/isctl/cmd/output.go:90 +0x570
main.GetCommands.func1.3.14.6.1(0x14000f5d680?, {0x14000116140, 0x1, 0x5})
	/go/src/github.com/cgascoig/isctl/cmd/cli.go:266756 +0x240
github.com/spf13/cobra.(*Command).execute(0x14000f5d680, {0x140001160f0, 0x5, 0x5})
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x4ac
github.com/spf13/cobra.(*Command).ExecuteC(0x140001bc780)
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x354
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
main.main()
	/go/src/github.com/cgascoig/isctl/cmd/main.go:105 +0xa00

--- a/cmd/output.go
+++ b/cmd/output.go
@@ -85,7 +85,11 @@ func resultHandler(result interface{}, httpResponse *http.Response, err error, o
 		if jsonPathFilter != "" {
 			log.Warn("Using -o jsonpath and --jsonpath together may produce unpredictable results")
 		}
-		printResultJSONPath(result, outputConfigParts[1])
+		if len(outputConfigParts) == 2 {
+			printResultJSONPath(result, outputConfigParts[1])
+		} else {
+			printResultJSONPath(result, "")
+		}
 
 	default:
 		printResultDefault(result)

